<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTcpServerControlWidget</class>
 <widget class="QWidget" name="UTcpServerControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="labelServerAddress">
     <property name="text">
      <string>Адрес сервера</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServerAddress"/>
   </item>
   <item>
    <widget class="QLabel" name="labelServerPort">
     <property name="text">
      <string>Порт сервера</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServerPort"/>
   </item>
   <item>
    <widget class="QLabel" name="labelServerLogin">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServerLogin"/>
   </item>
   <item>
    <widget class="QLabel" name="labelServerPassword">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditServerPassword"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
